### PR TITLE
Disable serial image pulls by default

### PR DESCRIPTION
**Pull request by aveshagarwal**
@ncdc @deads2k @liggitt @smarterclayton 

Still testing though.

### CODE REVIEW

**Comment by smarterclayton** [comment]
approved

File: pkg/cmd/server/kubernetes/node_config.go [hunk 1]
```diff
@@ -148,6 +148,7 @@ func BuildKubernetesNodeConfig(options configapi.NodeConfig) (*NodeConfig, error
 	server.PodInfraContainerImage = imageTemplate.ExpandOrDie("pod")
 	server.CPUCFSQuota = true // enable cpu cfs quota enforcement by default
 	server.MaxPods = 110
+	server.SerializeImagePulls = false // disable serial image pulls by default
```
Comment (ncdc) [inline pkg/cmd/server/kubernetes/node_config.go]: This only affects when you're running the kubelet that is a symlink to /usr/bin/origin. Also need to update the appropriate origin files too. @liggitt which ones make the most sense - node_options.go? node_args.go?

Comment (ncdc) [inline pkg/cmd/server/kubernetes/node_config.go]: Whoops, didn't follow the chain all the way up. This does apply to origin nodes too.

Comment (liggitt) [inline pkg/cmd/server/kubernetes/node_config.go]: not sure what you mean... this will effectively change the default for `openshift start node`

Comment (ncdc) [inline pkg/cmd/server/kubernetes/node_config.go]: Yeah, I thought this was only for kubelet. I stand corrected!

**Comment by ncdc** [comment]
[test]

**Comment by ncdc** [comment]
[test]

**Comment by ncdc** [comment]
Need to wait until after 1.2 is cut

**Comment by smarterclayton** [comment]
Why does this need to wait?

**Comment by smarterclayton** [comment]
[test]

**Comment by ncdc** [comment]
We have it in Trello for 3.3

**Comment by ncdc** [comment]
It also hasn't been QE'd for 3.2

**Comment by ncdc** [comment]
[test]

**Comment by smarterclayton** [comment]
It's time to merge this.

**Comment by ncdc** [comment]
Go go gadget [test]

**Comment by ncdc** [comment]
`Error: image cache/mysql:pullthrough not found` flake
[test]

**Comment by ncdc** [comment]
[merge]

**Comment by ncdc** [comment]
```
Running test/cmd/images.sh:131: executing 'oc import-image mysql --from=mysql --all --confirm' expecting success and text 'sha256:'...
FAILURE after 30.098s: test/cmd/images.sh:131: executing 'oc import-image mysql --from=mysql --all --confirm' expecting success and text 'sha256:': the command returned the wrong error code; the output content test failed
There was no output from the command.
Standard error from the command:
Error from server: Timeout: request did not complete within allowed duration
```

[merge]

**Comment by smarterclayton** [comment]
I love that we fix pullthrough and start getting timeouts.

**Pull request merged**